eng/feeds/arm/tspconfig.yaml

```diff
@@ -6,6 +6,39 @@ options:
     emitter-output-dir: "{project-root}/.."
     azure-resource-provider-folder: "resource-manager"
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/openapi.json"
+  "@azure-tools/typespec-csharp":
+    package-dir: "{package-name}"
+    clear-output-folder: true
+    model-namespace: false
+    namespace: "{package-dir}"
+  "@azure-tools/typespec-python":
+    package-dir: "{package-dir}"
```

Comment:
Should we differentiate between the placeholder value and the real value? `package-dir` in line#15 and line#13 are different cases. The former one is a placeholder, and users need to update it, but the latter one is the actual value, and users don't need to change it.

Reply:
so both properties will have at the end the same value, right? if one is a placeholder value, then why do we expose it to users?

Reply:
My understanding is that `csharp` namespace option here simply reuses the value of 'package-dir' option. Generally, exposing it allows user to specify the namespace. 
However, for python, as shown on line#15, the 'package-dir' option must be a concrete value and cannot be literally set as "{package-dir}".

Reply:
@msyyc could you help review here and add the values that should be included?

Reply:
Here is the example template: https://github.com/Azure/azure-rest-api-specs/blob/175d71800b942faaa4113416d5e395b8ba19ce31/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml#L19-L23.

As Ray said, `package-dir` shall be set with specific value while `package-name` can reuse it with `{package-dir}` so we shall change the format (maybe `package-dir: "<Your Package Dir>"` to guide users to set the specific value they want.

Reply:
@maririos , as requested by Allen, I updated the values to catch up the release train.